### PR TITLE
Micro mech code 

### PR DESCRIPTION
**Pull request by kcin2001**
DO NOT MERGE JUST PREPARING THIS ONE FOR ONCE I GET THE SPRITES
adds in construction paths for the 3 current micro mechs along with the parts 
also adds the ability to build the equipment for micro mechs (req tech and rnd levels can change upon suggestion)
adds in the crates for the relevant circuits along with adding them to the circuit printer
the gopher (mining micro mech) has the same building steps as the ripley and the 2 security micro mechs (gopher and weasel) are the same as the durand or gygax but need plasteel instead of the specific armor piece with the gopher needing more than the weasel
i should really change how much material is needed to build them too but i dont know how much to take off so ill think about it tommorow

### CODE REVIEW

File: code/game/mecha/micro/micro_equipment.dm [hunk 1]
```diff
@@ -98,7 +98,7 @@
 
 /obj/item/mecha_parts/mecha_equipment/tool/drill/micro
 	w_class = ITEMSIZE_LARGE
-	name = "drill"
+	name = "Miniture Drill"
```
Comment (Screemonster) [inline code/game/mecha/micro/micro_equipment.dm]: Should be "miniature"

**Comment by Screemonster** [comment]
What are the material costs of the non-armour parts compared to the gygax/ripley/durand? They should ideally be a lot lower than their corresponding bigmechs.

**Comment by kcin2001** [comment]
@Screemonster do you mean the actual body parts of the mech?

**Comment by Cameron653** [comment]
The design's materials requirements, I presume. Like
materials = list(DEFAULT_WALL_MATERIAL = 30000, "glass" = 11250)

**Comment by kcin2001** [comment]
i havent touched them yet so yeah they are the same as the other mechs

**Comment by Screemonster** [comment]
I'mma say just cut them all in half for now otherwise there's literally no reason to build a micromech over a full-size one.

**Comment by Arokha** [comment]
Just gonna slide this over here. If you finish it, reopen it.

**Pull request closed without merging**